app/views/layouts/application.html.erb

```diff
@@ -5,12 +5,25 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <%= render partial: "shared/cdn_assets" %>
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
   </head>
 
   <body>
-    <%= yield %>
+    <%= render partial: "shared/navbar" %>
+
+     <div class="container">
```

Comment:
this could be simpler. `<%= render "shared/flash" %>` then just implement notice/alert in app/shared/_flash.html.erb. Here is a good example https://gist.github.com/heratyian/1c556c443fd9c76461fa4b931acf70c3